cartographer/common/math.h

```diff
@@ -82,21 +82,20 @@ T atan2(const Eigen::Matrix<T, 2, 1>& vector) {
   return ceres::atan2(vector.y(), vector.x());
 }
 
-// Computes (A/'scale')^{-1/2} for A being symmetric, positive-semidefinite.
+// Computes 'A'^{-1/2} for A being symmetric, positive-semidefinite.
```

Comment:
As discussed, the constraint struct member is called 'sqrt_Lambda_ij' which should serve as this reference.